src/hooks/screen/useChatScreenHook.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import 'react-native-get-random-values';
+
+import * as React from 'react';
+import SimpleToast from 'react-native-simple-toast';
+import {v4 as uuid} from 'uuid';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider using a more secure random number generator. `uuid` is not cryptographically secure and may lead to predictable values, which could be a security risk if these IDs are used in a security-sensitive context.
```diff
- import {v4 as uuid} from 'uuid';
+ import {randomBytes} from 'crypto';
+ const uuid = () => randomBytes(16).toString('hex');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->